cmd/infracost/testdata/help/help.golden

```diff
@@ -18,14 +18,14 @@ EXAMPLES
       infracost breakdown --path /path/to/code --terraform-plan-flags "-var-file=my.tfvars"
 
 AVAILABLE COMMANDS
-  breakdown   Show full breakdown of costs
-  comment     Post an Infracost comment to GitHub, GitLab, Azure Repos or Bitbucket
-  completion  Generate shell completion script
-  configure   Display or change global configuration
-  diff        Show diff of monthly costs between current and planned state
-  help        Help about any command
-  output      Combine and output Infracost JSON files in different formats
-  register    Register for a free Infracost API key
+  breakdown        Show full breakdown of costs
```

Comment:
Any idea why this changes? Is it a cobra bug that still considers hidden commands when indenting?

Reply:
I tried to figured out that too, apparently it is a cobra bug.

Reply:
Yep, looks like a cobra bug.